aspnetcore/blazor/hosting-models.md

```diff
@@ -23,7 +25,7 @@ In a traditional server-rendered app, opening the same app in multiple browser s
 
 On the client, the Blazor script (`blazor.server.js`) establishes the SignalR connection with the server. The script is served to the client-side app from an embedded resource in the ASP.NET Core shared framework. The client-side app is responsible for persisting and restoring app state as required. 
 
-The Blazor Server hosting model offers several benefits:
+You should use the Blazor Server hosting model for the following benefits:
```

Comment:
```suggestion
The Blazor Server hosting model provides the following benefits:
```

Reply:
The goal is to enhance with the keywords/phrases that we need. This and the next one adds "should use" because the search phrase under discussion that we'd like to catch is ...

"should I use Blazor Server or Blazor {WebAssembly|WASM}"